source/agora/utils/TxBuilder.d

```diff
@@ -646,21 +804,21 @@ unittest
 /// Test with unfrozen remainder
 unittest
 {
-    const result = TxBuilder(GenesisBlock.payments.front)
-        .draw(Amount.UnitPerCoin * 50_000, WK.Keys.byRange.map!(k => k.address).take(3))
+    auto fee_rate = Amount(900);    // Using higher than min fee rate
```

Comment:
If this is meant to test that `TxBuilder` handles fees correctly, it should be its own test.

Reply:
ok will create extra test.